pkg/postgres/pgutils/retry.go

```diff
@@ -52,6 +57,7 @@ func Retry3[T any, U any](fn func() (T, U, error)) (T, U, error) {
 	for {
 		select {
 		case <-expirationTimer.C:
+			debug.PrintStack()
```

Comment:
Should only do in debug builds not release builds

Reply:
Doesn't really hurt in release builds as we're fataling anyway